test/com/google/javascript/jscomp/CommandLineRunnerTest.java

```diff
@@ -2460,6 +2460,33 @@ public void testWarningGuardQuotedValue() {
     testSame("alert('hello world')");
   }
 
+  @Test
+  public void testOutputSourceMapContainsSourcesContent() {
+    String inputString = "[{\"src\": \"\", \"path\": \"base.js\"},"
+        + "{\"src\": \"alert('foo');\", \"path\":\"foo.js\"}]";
+    args.add("--json_streams=BOTH");
+    args.add("--chunk=m1:1");
+    args.add("--chunk=m2:1:m1");
+    args.add("--source_map_include_content");
+
+    CommandLineRunner runner =
+        new CommandLineRunner(
+            args.toArray(new String[] {}),
+            new ByteArrayInputStream(inputString.getBytes(UTF_8)),
+            new PrintStream(outReader),
+            new PrintStream(errReader));
+
+    lastCompiler = runner.getCompiler();
+    try {
+      runner.doRun();
+    } catch (IOException e) {
+      e.printStackTrace();
+      assertWithMessage("Unexpected exception " + e).fail();
+    }
+    String output = new String(outReader.toByteArray(), UTF_8);
+    assertThat(output).contains("\\\"sourcesContent\\\":");
```

Comment:
Don't we care what value is associated with "sourcesContent"?
Would it be reasonable to assert on the exact content of the output?
As a reader not intimately familiar with what the output *should* look like,
it would help a lot for the test to tell me what it should look like.

I realize it would be more prone to "spurious" breakages, but those would be pretty easy to fix when they happen.

Reply:
I was kind of assuming that source maps in general were tested elsewhere. The issue was that with code splitting, only the first emitted chunk had any sources content. I just really wanted to ensure it was present.

That being said though, I can add the exact snippet there.

Reply:
Added.